AFQ/tractography.py

```diff
@@ -117,38 +141,58 @@ def track(params_file, directions="det", max_angle=30., sphere=None,
     elif odf_model == "CSD":
         dg = dg.from_shcoeff(model_params, max_angle=max_angle, sphere=sphere)
 
-    if stop_mask is None:
-        stop_mask = np.ones(params_img.shape[:3])
+    if tracker == "local":
+        if stop_mask is None:
+            stop_mask = np.ones(params_img.shape[:3])
 
-    if stop_mask.dtype == 'bool':
-        stopping_criterion = ThresholdStoppingCriterion(stop_mask,
-                                                        0.5)
-    else:
-        stopping_criterion = ThresholdStoppingCriterion(stop_mask,
-                                                        stop_threshold)
+        if stop_mask.dtype == 'bool':
+            stopping_criterion = ThresholdStoppingCriterion(stop_mask,
+                                                            0.5)
+        else:
+            stopping_criterion = ThresholdStoppingCriterion(stop_mask,
+                                                            stop_threshold)
+
+        my_tracker = VerboseLocalTracking
+    elif tracker == "pft":
+        pve_wm_data, pve_gm_data, pve_csf_data = stop_threshold
+        my_tracker = VerboseParticleFilteringTracking        
+        if stop_mask == "CMC":
+            average_voxel_size = np.mean(params_img.header.get_zooms()[:3])
+            stopping_criterion = CmcStoppingCriterion.from_pve(
+                pve_wm_data,
+                pve_gm_data,
+                pve_csf_data,
+                step_size=step_size,
+                average_voxel_size=average_voxel_size)
```

Comment:
I now realize that this needs to be the voxel size of the PVE maps. Need to think how we get that in there.

Reply:
Maybe instead of a tuple, it's its own class:

seed_mask = PFTInfo(afm.MaskFile(f"WM_probseg"), afm.MaskFile(f"GM_probseg"), afm.MaskFile(f"CSF_probseg"), "ACT")

The API would interact with it normally, but the track function would recognize it and access parameters like average_voxel_size which would have been calculated when the masks were loaded.